### PR TITLE
Add currentUser to group creation

### PR DESCRIPTION
**Pull request by RRicardotj**
This PR add current user id authenticated when a group is created

### CODE REVIEW

File: src/controllers/group/index.js [hunk 1]
```diff
@@ -30,7 +30,11 @@ export const getParticipantsCount = (obj, args, context) => {
 export const createGroup = async (obj, { input }, context) => {
   const productId = await createProductIfNotGiven(input, context.models)
 
-  return context.models.group.create({ ...input, productId })
+  return context.models.group.create({
+    ...input,
+    productId,
+    sellerId: context.currentUser && context.currentUser.id
```
Comment (abelosorio) [inline src/controllers/group/index.js:36]: We shouldn't ask if the currentUser exists. This mutation can NOT be executed if there is no logged-in user. Let's wrap this resolver inside a requireAuth hoc ;)

File: src/schema/resolvers/group.js [hunk 2]
```diff
@@ -11,7 +12,7 @@ const resolvers = {
     group: groupController.getGroupById
   },
   Mutation: {
-    createGroup: groupController.createGroup
+    createGroup: (obj, params, context) => requireAuth(obj, params, context)(groupController.createGroup)
```
Comment (abelosorio) [inline src/schema/resolvers/group.js:15]: Let's refactor this to achieve more simplicity: `requireAuth(groupController.createGroup)`

Comment (RRicardotj) [reply src/schema/resolvers/group.js:15]: @abelosorio  but how `requireAuth` access to `context` without passing parameters?

Comment (abelosorio) [reply src/schema/resolvers/group.js:15]: When you do this:
```javascript
createGroup: requireAuth(groupController.createGroup)
```

The _output_ of `requiredAuth` will be called like any other controller ;) So the magic there, is that requiredAuth should return a function:

```javascript
requiredAuth = (obj, params, context) => {
  return (controller) => {
    if (context.currentUser) // ...
    // ...
  }
}
```

Comment (abelosorio) [reply src/schema/resolvers/group.js:15]: Sorry:
```javascript
requiredAuth = (controller) => {
  return (obj, params, context) => {
    if (context.currentUser) // ...
    // ...
  }
}
```

Comment (RRicardotj) [reply src/schema/resolvers/group.js:15]: Ummm ok ok

**Pull request merged**